### PR TITLE
wx no longer compplains of unsupported languages

### LINKED ISSUE #8064: Loading NVDA with specific languages shows "Locale not supported by OS" warning message box
Hello folks! When changing the language to Kirgyz, NVDA works Properly till the restart. After that, the following error message occurs:

---------------------------
Nvda Warning
---------------------------
Locale 'Kirghiz' not supported by OS.
---------------------------
OK   
---------------------------
Steps to reproduce:
1. Open the NVDA menu.
2. Go to Preferences, General Settings.
3. Choose and select Kirgyz language from the languages list. After the restart, the error I'm talking about occurs.

NVDA version: 2018.1
Operating system: Windows 10 Fall Creators Update 64-bit.
Hope this helps.

### PR DESCRIPTION
**Pull request by michaelDCurran**
### Link to issue number:
Fixes #8064 

### Summary of the issue:
There are some languages NVDA has a translation for but WX itself does not. An example of this is Mongolian (mn).
When switching to one of these languages, WX shows a warning dialog on NvDA startup stating that  the language is not supported by the OS.
 
### Description of how this pull request fixes the issue:
Core now checks to see if wxPython actually supports the language (I.e. has a translation database for it), and only switches to the language if it does.

### Testing performed:
Started NVDA in Mongolian: NVDA started, and the warning dialog did not appear. NVDA messages were translated, but there was no WX translation obviously.
Started NVDA in French: NVDA interface was translated in French and so too was wx's controls.

### Known issues with pull request:
None.

### Change log entry:
Bug fixes:
* When NVDA is set to languages such as Kirgyz, Mongolian or Macedonian, it no longer shows a dialog on start-up warning that  the language is not supported by the Operating System.

### CODE REVIEW

**Review by josephsl — DISMISSED** [review]
Hi,

I see. I think we need to alert translators of this fact soon so they understand what's going on.

Thanks.

File: source/core.py [hunk 1]
```diff
@@ -368,6 +368,11 @@ def handlePowerStatusChange(self):
 		wxLang=locale.FindLanguageInfo(lang.split('_')[0])
 	if hasattr(sys,'frozen'):
 		locale.AddCatalogLookupPathPrefix(os.path.join(os.getcwdu(),"locale"))
+	# #8064: Wx might know the language, but may not actually contain a translation database for that language.
+	# If we try to initialize this language, wx will show a warning dialog.
+	# Therefore treat this situation like wx not knowing the language at all.
+	if not locale.IsAvailable(wxLang.Language):
+		wxLang=None
```
Comment (leonardder) [inline source/core.py]: It might make sense to log this.

**Comment by michaelDCurran** [comment]
I think it does get logged, further down. The 'else' clause for if wxLang.

I don't think we need a specific log to differentiate between 
findLanguageInfo returning nothing and isAvailable returning False as 
both clearly mean there is no translation.

**Comment by leonardder** [comment]
Ok, I'm fine with that.

**Pull request merged**